src/Symfony/Component/VarDumper/Tests/Dumper/CliDumperTest.php

```diff
@@ -46,7 +46,7 @@ public function testGet()
         $intMax = PHP_INT_MAX;
         $res = (int) $var['res'];
 
-        $r = defined('HHVM_VERSION') ? '' : '#%d';
+        $r = '#%d';
```

Comment:
I would put it directly in the string instead of using a variable, as it does not change anymore